.github/renovate.json

```diff
@@ -1,20 +1,27 @@
 {
-  "schedule": ["every weekend"],
-  "labels": ["dependencies"],
+  "extends": [ "config:base", ":dependencyDashboardApproval" ],
```

Comment:
> `dependencyDashboardApproval`

Makes PRs have to be approved on the Renovate dashboard issue before they will be created.

I'm not sure we're keeping up with the dashboard at core much, makes me wonder if we made the right call or were too conservative going for a dashboard?

Then again, with how heavily grouped the PRs are configured lower in the file, maybe it's for the best not to have these auto-opend all the time?

Trade-offs... Hmmb.

Reply:
Yeah trade offs for sure. It is something I keep up on (ish) to at least look at where some of our problems might be, or what are some low hanging fruit to bump.

But really I think that's what we may need to stick with, since I know when originally doing the poll to even setup automated dep management the biggest blocker and promise was that we wouldn't let things go wild just making PRs everyday. Honestly even Renovate's automated security PRs can be annoying.

Reply:
True, and thanks for setting that up and keeping up with things over there at core with it.